requirements.txt

```diff
@@ -0,0 +1,4 @@
+django
+djangorestframework
+beautisoup4
+requests
```

Comment:
No need to create requirements.txt manually. 
Use pip freeze > requirements.txt

Reply:
Ok